pyramid_favicon/favicon.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""Serve the favicon.ico at the application root."""
+
+from pyramid.response import FileResponse
+from pyramid.security import NO_PERMISSION_REQUIRED
+from pyramid.view import view_config
+
+import os
+
+
+@view_config(
+    route_name='favicon',
+    permission=NO_PERMISSION_REQUIRED,
+)
+def favicon(request):
+	
+	here = os.path.dirname(__file__)
+	
+	settings = request.registry.settings
+	favicon_folder = settings.get('favicon_folder', None)
```

Comment:
Minor nitpick: this should be named favicon_path.

You could have favicon_folder, but then you would also need favicon_filename, or have filename hardcoded to ``favicon.ico``. So using path is the best approach because you provide folder and filename in one variable.